locale/pt_BR/manager.po

```diff
@@ -171,13 +171,13 @@ msgid "manager.people.enrollSyncJournal"
 msgstr "na revista"
 
 msgid "manager.people.mergeUsers.from.description"
-msgstr "Escolha um usuário para mesclar seu cadastro dentro de outra conta (quando um usuário possui duas contas distintas, em vez de acumular funções editoriais). O cadastro selecionado primeiro será excluído e suas submissões e histórico de tarefas atribuías ao segundo cadastro escolhido."
+msgstr "Escolha um usuário para mesclar seu cadastro dentro de outra conta (quando um usuário possui duas contas distintas, em vez de acumular funções editoriais). O cadastro selecionado primeiro será excluído e suas submissões e histórico de tarefas atribuídas ao segundo cadastro escolhido."
 
 msgid "manager.people.mergeUsers.into.description"
 msgstr "Escolha um usuário a quem deseja atribuir as submissões e histórico de tarefas do cadastro anterior."
 
 msgid "manager.people.roleEnrollment"
-msgstr "Papel de {$role}"
+msgstr "Papel dos usuários"
```

Comment:
@Lu-Brito, the English translation does make use of the `{$role}` variable -- that should be maintained for consistency with other translations.

Reply:
@asmecher I already did the changes, but I couldn't do the pull request. Could you help me? How can I do it? Thank you!

Reply:
Usually you'd commit the change to your existing branch, then push your branch up to your Github account. That should automatically add the change to this PR.

Reply:
So, I'm asking exactly why I've already committed the change to my repository and the changes didn't automatically appear in PR. Or did they show up? Thank you @asmecher!

Reply:
@Lu-Brito, it does show up -- as a second commit in the same PR: https://github.com/pkp/ojs/pull/2563/commits/92d375865ca0a44698e8ef3f015abc06ee90982b
I'll merge this, thanks!